howtos/sanitizers.rst

```diff
@@ -155,15 +155,17 @@ flags to the build. It could be something like:
 .. code-block:: python
     :caption: *sanitizer_hook.py*
 
-    def set_sanitize_address_flag(self):
-        self._old_cxx_flags = os.environ.get("CXXFLAGS")
-        os.environ["SOURCE_DATE_EPOCH"] = _old_flags + " -fsanitize=address"
+    _OLD_CXX_FLAGS = None
 
-    def reset_sanitize_address_flag(self):
-        if self._old_cxx_flags is None:
+    def set_sanitize_address_flag():
```

Comment:
It is not totally evident why the ``self.old_cxx_flags`` doesn't work, is it because it is called from different instances of the hook?

Reply:
What is `self._old_cxx_flags`?
It doesn't seem, that we're in some class method here, so `self` doesn't make sense for me.
That's why we have to somehow save the state between function calls, that's why I used global variable as the simplest way to do so.